doc/en/components/layouts/dock-manager-customization.md

```diff
@@ -0,0 +1,104 @@
+
+### Outer Docking
+To perform outer dock, you must first meet specific criteria. In Dock Manager, outer docking is only allowed within a document host, meaning you will be docking the target pane to the outer regions of the document host.
+
+In order to perform an outer top dock for example, must fall within the region defined by the top border of the document host and stay within the top border offset. The offset distance is specified as a proximity dock outer threshold and is explicitly set to a value of 25px. The proximity dock threshold's value of 50px remains unchanged which essentially means that in this scenario, only a 25px area is available for inner docking. In cases where there is a splitter inside the document host, outer dock can be performed from both sides of the splitter.
+
+> NOTE: When proximity dock is enabled, users would not be able to perform an outer dock for both left and right and top and bottom positions from a single pane. For example in a scenario where two split panes are present with a splitter between them, users will be able to perform only an outer left dock when the dragged pane is over the left split pane and vice versa.
+
+Try it for yourself in the sample below:
+
+`sample="/layouts/dock-manager/proximity-dock", height="600", alt="{Platform} Dock Manager Proximity Dock Example"`
+
+## Focus Panes Programmatically
+
+The `focusPane` method allows developers to dynamically and programmatically focus a specific pane within the layout by providing the `contentId` of the desired pane.
+
+```ts
+this.dockManager.focusPane('content1');
+```
+
+When using the `focusPane` method, the behavior varies depending on the state of the targeted pane. Here's how it works:
+
+1. **Floating Panes**: If the pane is floating, invoking `focusPane` will bring it into focus, set it as the activePane and ensure it appears on top of any other floating panes.
+
+2. **Unpinned Panes**: When the targeted pane is unpinned, `focusPane` will flyout the pane to its open state.
+
+3. **Regular Pinned Panes**: For regular pinned panes, `focusPane` will set the pane as the `activePane`.
+
+
+By utilizing this method, developers can effortlessly control the visibility and positioning of panes based on user interaction or application events.
+
+Try it for yourself in the sample below:
+
+`sample="/layouts/dock-manager/focus-panes", height="600", alt="{Platform} Focus panes programmatically in Dock Manager Example"`
+
+
+## Auto-hide Pane Headers
+
+With the `showPaneHeaders` property of the DockManager, developers now have the flexibility to control the visibility of pane headers within the `layout`. By default, `showPaneHeaders` is set to **true** ensuring that pane headers are always visible. When set to **false**, all pane headers will be hidden until you hover your mouse over the top edge of a content pane. The corresponding pane header will appear and it will smoothly hide once your mouse moves away. Take a look at the example below: 
```

Comment:
Update this section - `showPaneHeaders` was later modified to be either `'onHoverOnly'` or `'always'`